manage-data/data-store/data-streams/set-up-tsds.md

```diff
@@ -150,79 +156,105 @@ PUT _index_template/my-weather-sensor-index-template
 { "create":{ } }
-{ "@timestamp": "2099-05-06T16:25:42.000Z", "sensor_id": "SYKENET-000001", "location": "swamp", "temperature": 32.4, "humidity": 88.9 }
+{ "@timestamp": "2099-05-06T16:25:42.000Z", "sensor_id": "SENSOR-002", "location": "warehouse-B", "temperature": 32.4, "humidity": 88.9 }
+```
 
-POST metrics-weather_sensors-dev/_doc
+```console
+POST metrics-weather-sensors/_doc
 {
   "@timestamp": "2099-05-06T16:21:15.000Z",
-  "sensor_id": "SYKENET-000001",
-  "location": "swamp",
+  "sensor_id": "SENSOR-00002",
+  "location": "warehouse-B",
   "temperature": 32.4,
   "humidity": 88.9
 }
 ```
+::::
+
+::::{step} Verify setup
+To make sure your time series data stream is working, try some GET requests.
 
-You can also manually create the TSDS using the [create data stream API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-create-data-stream). The TSDS’s name must still match one of your template’s index patterns.
+View data stream details:
 
 ```console
-PUT _data_stream/metrics-weather_sensors-dev
+GET _data_stream/metrics-prod 
 ```
 
+Check the document count in a time series data stream:
 
-## Secure the TSDS [secure-tsds]
+```console
+GET metrics-prod/_count 
+```
 
-Use [index privileges](elasticsearch://reference/elasticsearch/security-privileges.md#privileges-list-indices) to control access to a TSDS. Granting privileges on a TSDS grants the same privileges on its backing indices.
+Query the time series data:
 
-For an example, refer to [Data stream privileges](../../../deploy-manage/users-roles/cluster-or-deployment-auth/granting-privileges-for-data-streams-aliases.md#data-stream-privileges).
+```console
+GET metrics-prod/_search 
+{
+  "size": 5,
+  "sort": ["@timestamp"]
+}
+```
 
 
-## Convert an existing data stream to a TSDS [convert-existing-data-stream-to-tsds]
+::::
 
-You can also use the above steps to convert an existing regular data stream to a TSDS. In this case, you’ll want to:
 
-* Edit your existing index lifecycle policy, component templates, and index templates instead of creating new ones.
-* Instead of creating the TSDS, manually roll over its write index. This ensures the current write index and any new backing indices have an [`index.mode` of `time_series`](time-series-data-stream-tsds.md#time-series-mode).
+## Advanced setup
 
-    You can manually roll over the write index using the [rollover API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-rollover).
+### Convert an existing data stream to a TSDS [convert-existing-data-stream-to-tsds]
 
-    ```console
-    POST metrics-weather_sensors-dev/_rollover
-    ```
+You can convert an existing regular data stream to a TSDS. Follow these steps:
 
+1. Update your existing index template to include time series settings. Also update your index lifecycle policy (if any) and component templates (if any). 
```

Comment:
The second sentence here is a vague, when I read I have no idea what kind of updates it is talking about. Maybe something like:

> Update your existing index template and/or component templates (if any) to include time series settings.

It is reasonable to bundle the index template and component templates together, because depending on the setup it might be enough to update only one component template. 

About the ILM policy, I have no idea what updates it is referring to, that's why I would suggest to remove it.